problems/4.arrays.js

```diff
@@ -2,27 +2,38 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  let arr = []
+  for (i = 0; i < arr.length; i++){
```

Comment:
NO loop needed. 
First element is arr[0] , last is arr[arr.length - 1] Average would be those numbers added together and divided by 2.